microsoft-edge/webview2/release-notes/index.md

```diff
@@ -56,7 +56,7 @@ Background: If you're building a Windows Presentation Foundation (WPF) app and u
 This issue occurs because the WPF control uses the WPF `HwndHost` to host the Win32 WebView2 control, and `HwndHost` has an issue with airspace; see [Mitigating Airspace Issues In WPF Applications](https://dwayneneed.github.io/wpf/2013/02/26/mitigating-airspace-issues-in-wpf-applications.html).
```

Comment:
2x in relnotes: move Mitigating link into See Also list, like in new WPF plat article.